internal/workspace_client_test.go

```diff
@@ -0,0 +1,15 @@
+package internal
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUcAccWorkspaceClient_CurrentWorkspaceId(t *testing.T) {
```

Comment:
We use UC acceptance tests because it's the only environment where the current workspace ID is configured.